udata_gouvfr/models.py

```diff
@@ -29,9 +29,11 @@
 C3 = 'c3'
 NECMERGITUR = 'nec'
 OPENFIELD16 = 'openfield16'
+SPD = 'spd'
 Dataset.__badges__[C3] = _('C³')
 Dataset.__badges__[NECMERGITUR] = _('Nec Mergitur')
 Dataset.__badges__[OPENFIELD16] = 'Openfield 16'
+Dataset.__badges__[SPD] = _('Public Data Service')
```

Comment:
Is that the way it is translated? If no translation is possible, let's keep it in French.